src/sbl_filing_api/entities/models/dao.py

```diff
@@ -12,34 +12,22 @@ class Base(AsyncAttrs, DeclarativeBase):
     pass
 
 
-class SubmitterDAO(Base):
-    __tablename__ = "submitter"
+class UserActionDAO(Base):
+    __tablename__ = "user_action"
     id: Mapped[int] = mapped_column(index=True, primary_key=True, autoincrement=True)
-    submission: Mapped[int] = mapped_column(ForeignKey("submission.id"))
-    submitter: Mapped[str]
-    submitter_name: Mapped[str] = mapped_column(nullable=True)
-    submitter_email: Mapped[str]
-
-
-class AccepterDAO(Base):
-    __tablename__ = "accepter"
-    id: Mapped[int] = mapped_column(index=True, primary_key=True, autoincrement=True)
-    submission: Mapped[int] = mapped_column(ForeignKey("submission.id"))
-    accepter: Mapped[str] = mapped_column(nullable=True)
-    accepter_name: Mapped[str] = mapped_column(nullable=True)
-    accepter_email: Mapped[str]
-    acception_time: Mapped[datetime] = mapped_column(server_default=func.now())
-
-    def __str__(self):
-        return f"Acception ID: {self.id}, Accepter: {self.accepter}, Accepter Name: {self.accepter_name}, Accepter Email: {self.accepter_email}, Submission: {self.submission} Acception: {self.acception_time}"
+    user_id: Mapped[str]
+    user_name: Mapped[str]
+    user_email: Mapped[str]
+    action_type: Mapped[UserActionType] = mapped_column(SAEnum(UserActionType))
+    timestamp: Mapped[datetime] = mapped_column(server_default=func.now())
 
 
 class SubmissionDAO(Base):
     __tablename__ = "submission"
     id: Mapped[int] = mapped_column(index=True, primary_key=True, autoincrement=True)
     filing: Mapped[int] = mapped_column(ForeignKey("filing.id"))
-    submitter: Mapped[SubmitterDAO] = relationship("SubmitterDAO", lazy="joined")
-    accepter: Mapped[AccepterDAO] = relationship("AccepterDAO", lazy="joined")
+    submitter: Mapped[int] = mapped_column(ForeignKey("user_action.id"))
+    accepter: Mapped[int] = mapped_column(ForeignKey("user_action.id"), nullable=True)
     state: Mapped[SubmissionState] = mapped_column(SAEnum(SubmissionState))
```

Comment:
For these, I think we're going to also want a relationship so we can get full the UserAction object back to the frontend.  We do something similar in the user-fi repo, for instance:
```
    primary_federal_regulator_id: Mapped[str] = mapped_column(ForeignKey("federal_regulator.id"), nullable=True)
    primary_federal_regulator: Mapped["FederalRegulatorDao"] = relationship(lazy="selectin")
```
So that the frontend will get more than just the UserAction id.